algorithms/schema-free/betweenness_cent.gsql

```diff
@@ -7,11 +7,13 @@ CREATE QUERY Betweenness_cent(SET<STRING> v_type, SET<STRING> e_type, STRING re_
   Parameters:
   v_type: vertex types to traverse                 print_accum: print JSON output
   e_type: edge types to traverse                   result_attr: INT attr to store results to
+  re_type: reverse edge type in directed graph, in undirected graph set re_type=e_type
   max_hops: look only this far from each vertex    file_path: file to write CSV output to
   top_k: report only this many top scores          display_edges: output edges for visualization
+  
    */ 
-  TYPEDEF TUPLE<VERTEX Vertex_ID, FLOAT score> VertexScore;
-  HeapAccum<VertexScore>(top_k, score DESC) @@topScores;
+  TYPEDEF TUPLE<VERTEX Vertex_ID, FLOAT score> VertexScore; #tuple to store harmonic centrality score
```

Comment:
Why does this say harmonic centrality score and not betweenness centrality score?